	Otherwise fails on x86_64-solaris(10) due to (crazy?)
	system definition of registers.
	https://tug.org/pipermail/tlbuild/2020q1/004599.html
	https://tug.org/pipermail/tlbuild/2020q1/004599.html

--- source/common/ubidiimp.h~	2020-12-17 11:34:38.000000000 -0800
+++ source/common/ubidiimp.h	2021-02-13 08:20:09.076105678 -0800
@@ -42,4 +42,6 @@
     See UCharDirection in uchar.h .
 */
+#undef ES
+#undef CS
 enum {
     L=  U_LEFT_TO_RIGHT,                /*  0 */
